app/src/main/java/org/systers/mentorship/view/activities/AboutActivity.kt

```diff
@@ -1,41 +1,60 @@
 package org.systers.mentorship.view.activities
 
 import android.content.Intent
-import android.content.res.Resources
-import android.net.Uri
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
 import kotlinx.android.synthetic.main.activity_about.*
 import org.systers.mentorship.R
 
 class AboutActivity : AppCompatActivity() {
+    override fun onNavigateUp(): Boolean {
+        finish()
+        return super.onNavigateUp()
+    }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_about)
-        setTitle(R.string.fragment_title_about);
+        setTitle(R.string.fragment_title_about)
 
 
         supportActionBar?.setDisplayHomeAsUpEnabled(true)
 
         btnGit.setOnClickListener {
-
-            startActivity(Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.url_github))))
+            val intent = Intent(this, WebViewActivity::class.java)
+            intent.putExtra("URL", getString(R.string.url_github))
+            intent.putExtra("Name", "Github")
```

Comment:
Hey @AnuthaDev I think it's better to create these constants - "URL", "Name", "Github", "Slack", etc in **Constants.kt** file separately. It's the recommended approach. So, please add these constants in that file.